.storybook/config.js

```diff
@@ -50,6 +52,12 @@ addDecorator((storyFn) => (
 //redirect console error / logs / warns to action logger
 addDecorator((storyFn, context) => withConsole()(storyFn)(context));
 
+addDecorator((storyFn) => (
+    <I18nextProvider i18n={i18n}>
+        <Suspense fallback="Loading...">{storyFn()}</Suspense>
```

Comment:
I kept hearing whispers of React Suspense as a new feature coming out - guess it's time for me to read up on that 😄